.github/workflows/CI.yml

```diff
@@ -61,7 +61,7 @@ jobs:
         shell: julia --project=. --color=yes {0}
       - uses: julia-actions/julia-runtest@v1
       - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v2
+      - uses: codecov/codecov-action@v4
```

Comment:
Version 4 of the action requires a token, tokenless uploads are not supported anymore.

Reply:
I added a secret "CODECOV_TOKEN" to the repo and changed the workflow configuration.